src/prefect/experimental/workers/base.py

```diff
@@ -20,6 +20,18 @@
 from prefect.settings import PREFECT_WORKER_PREFETCH_SECONDS, get_current_settings
 from prefect.states import Crashed, Pending, exception_to_failed_state
 from prefect.utilities.dispatch import register_base_type
+from prefect.utilities.templating import apply_values, resolve_block_document_references
+
+
+class Unset:
```

Comment:
this is duplicated now in the utilities file, probably worth importing from there